docs/flashbots-core/releases/v0.2-release.md

```diff
@@ -0,0 +1,131 @@
+## Changelog
+
+#### `eth_sendBundle` RPC parameters change
+The specification of the `eth_sendBundle` RPC is changing to support new features of v0.2. This change must be adopted by all searchers in order to be compatible with the v0.2 relay.
+
+Here is the change in the specification:
+
+v0.1
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendBundle",
+  "params": [
+    signedTxs,    // Array[String], A list of signed transactions to execute in an atomic bundle
+    blockNumber,  // String, a hex encoded block number for which this bundle is valid on
+    minTimestamp, // (Optional) Number, the minimum timestamp for which this bundle is valid, in seconds since the unix epoch
+    maxTimestamp  // (Optional) Number, the maximum timestamp for which this bundle is valid, in seconds since the unix epoch
+  ]
+}
+```
+
+v0.2
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendBundle",
+  "params": [
+    {
+      signedTxs,         // Array[String], A list of signed transactions to execute in an atomic bundle
+      blockNumber,       // String, a hex encoded block number for which this bundle is valid on
+      minTimestamp,      // (Optional) Number, the minimum timestamp for which this bundle is valid, in seconds since the unix epoch
+      maxTimestamp,      // (Optional) Number, the maximum timestamp for which this bundle is valid, in seconds since the unix epoch
+      revertingTxHashes, // (Optional) Array[String], A list of tx hashes that are allowed to revert 
+    }
+  ]
+}
+```
```

Comment:
@RyRy79261 how do I use tabs for this? 

https://docusaurus.io/docs/markdown-features/code-blocks#multi-language-support-code-blocks